server/test/controllers/role.specs.js

```diff
@@ -10,12 +10,12 @@ dotenv.config();
 
 const superAdminToken = auth.setUserToken(mockData.superAdmin);
 const adminToken = auth.setUserToken(mockData.admin);
-const userToken = auth.setUserToken(mockData.regularUser);
-const noToken = auth.setUserToken(mockData.noToken);
+const userToken = auth.setUserToken(mockData.user);
 
-describe('Role Endpoints', () => {
+describe('Role controller', () => {
```

Comment:
I want to believe that this test is for much more that just your controller. Think about changing the language here such that it communicates the actual thing you are testing for.